### PR TITLE
Add support for different source repository/endpoint, user id mappings

### PR DESCRIPTION
**Pull request by kga**
We wanna migrate issues from GitHub Enterprise to github.com, so add support for set different access_token, endpoint and user id mappings.

### CODE REVIEW

File: migrate_issues.rb [hunk 1]
```diff
@@ -98,16 +109,18 @@ def hendl_issue(original)
...
+    table_code = table(mapped_login_id, "Original issue by @#{mapped_login_id} - #{table_link}")
     body = [table_code, original.body]
     data = {
       issue: {
         title: original.title,
         body: body.join("\n\n"),
         created_at: original.created_at.iso8601,
-        labels: actual_label + [tool_name_label],
+        assignee: original.assignee.nil? ? nil : map_login_id(original.assignee.login),
+        labels: actual_label,
```
Comment (maku693) [inline migrate_issues.rb]: 「インポートしました」的なラベルがついてると便利そうかなと思いました

Comment (kga) [reply migrate_issues.rb]: Sounds good!!!!

Comment (kga) [reply migrate_issues.rb]: https://github.com/kga/monorepo/pull/1/commits/12dffef4989e9252a4c3f132cc09b3b2c816d372 add `imported` label

**Comment by kga** [comment]
Thanks!!

**Pull request merged**